.circleci/config.yml

```diff
@@ -215,10 +207,6 @@ jobs:
           name: Run e2e preferences test
           command: |
             python -m scripts.run_e2e_tests --skip-install --skip-build --suite="preferences" --prod_env
-      - run:
-          name: Run e2e profileFeatures test
-          command: |
-            python -m scripts.run_e2e_tests --skip-install --skip-build --suite="profileFeatures" --prod_env
```

Comment:
If we are planning to move this test back to circleCi then I and Kevin has created PR to fix issues in this suite so maybe we can leave it to cicleCI? (Leaving it to you to decide I'm fine with either!)

Reply:
@DubeySandeep can you link to the PR that fixes the issues in this suite for reference? If the underlying issues are fixed, do we have any other advantages to moving to TravisCI @U8NWXD?

Reply:
Let's leave the subscriptions and profileFeatures tests in CircleCI so we can better evaluate whether they're fixed